manim/utils/color.py

```diff
@@ -114,7 +331,10 @@ def random_bright_color():
 
 
 def random_color():
-    return random.choice(list(COLOR_MAP.values()))
+    import time
+
+    random.seed(time.time())
```

Comment:
import time
random.seed(time.time())
are not really necessary here, but it would guarantee, that colors are every time new random colors when you run your manim script again.
Alternatively, one could add an example with random colors to the docs and say that you can change the starting behaviour with setting e.g. random.seed(42)

Reply:
I do't understand what you say. First you say that those lines are not necessary. Then you describe exactly what they are doing.

These lines are there precisely to guarantee that every time you run a script, the random colors are indeed random. This is intended behavior. If this is confusing or causing problems, we should discuss in some other issue.

Reply:
Ok, let me clarify:
* with random.seed(time.time())  random every time one runs the script.
* However, without random.seed(time.time()),  it would be easier to control and reproduce this random behaviour, therefore you would need to type into your scene e.g.
random.seed(42) or random.seed(43) ... and do this as long as you have a random color that you like

Reply:
Ok I see, thanks. So if the problem is that this doesn't allow the user to control the random seed, then I agree. Maybe we should open a new issue for this.

Reply:
I would also prefer to give the user the opportunity to choose the random seeds by themselves (I will add an example to the docs)
So I can remove the time seed here.
And as this function is really not that important, I think its not worth having a long discussion about it.

Reply:
Yeah ok just remove it here then.